src/Kros.Swagger.Extensions/SwaggerExtensions.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Swashbuckle.AspNetCore.Swagger;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Kros.Swagger.Extensions
+{
+    /// <summary>
+    /// Swagger service extensions.
+    /// </summary>
+    public static class SwaggerExtensions
+    {
+#pragma warning disable IDE1006 // Naming Styles
+        private const string SwaggerDocumentationSectionName = "SwaggerDocumentation";
+        private const string DefaultOAuthClientId = "kros_postman";
+#pragma warning restore IDE1006 // Naming Styles
+
+        /// <summary>
+        /// Registers Swagger documentation generator to IoC container.
+        /// </summary>
+        /// <param name="services">IoC container.</param>
+        /// <param name="configuration">Application configuration.</param>
+        public static IServiceCollection AddSwaggerDocumentation(this IServiceCollection services, IConfiguration configuration)
+        {
+            string assemblyName = AppDomain.CurrentDomain.FriendlyName;
+            Info swaggerDocumentationSettings = GetSwaggerDocumentationSettings(configuration);
+
+            if (swaggerDocumentationSettings != null)
```

Comment:
By my opinion will be better throw expcetion if configuration section is missing. The same in the `UseSwaggerDocumentation`.